### ISSUE TITLE
Create a Join Us page

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/website#752]
### Overview
We have multiple ways to bring volunteers into hack for LA besides projects, we need a page that provides a pathway for all types of contributions

### Action Items
- [x] Product
   - [x] Add general content to page to be augmented as needed
- [x] UI
   - [x] Mock up an about page that is of the same illustrative design quality as the rest of the site.
- [ ] Development
   - [x]  Create page based on Figma mockup
   - [ ]  Add to menu of HfLA site - desktop and mobile
   - [x] Add screenshots to issue to get visual signoff from product
- [ ] Product
   - [ ] Signoff
- [ ] Development
   - [ ] PR
- [ ] Team review

### Resources/Instructions
[Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=2991%3A0)

**Comment by yuikomajima** [comment]
Here's a pass at the Join us page using the icon you suggested. If we can consolidate the partner with us forms into 1 (by adding a question - are you gov't, non-profit etc.) then it could be cleaner without having 3 CTAs. https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3058%3A21055

**Comment by ExperimentsInHonesty** [comment]
The three forms are slightly different.  But I'm will to take a look again at the questions and see if we can't revise things

**Comment by alexandrastubbs** [comment]
@yuikomajima looks great! Could you play with the circles in the donation section so that they 1) take up a bit more space 2) illustrate the quantity we spend toward each thing (i.e. monthly = most spend, annual = middle spend, occasional = least spent)? Also maybe we change their color (green for money?) and the formatting of the section so that it is more consistent with the other modules.

**Comment by yuikomajima** [comment]
made the above changes. To make the format more consistent, it might be better to have the expense breakdown in text to not have 3 images in one module. https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3058%3A21055

**Comment by yuikomajima** [comment]
please see here for mobile view. Please review and update copy! https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3058%3A21055

**Comment by alexandrastubbs** [comment]
@yuikomajima all good, we updated copy and moved some things around. This is ready for development.

**Comment by trtincher** [comment]
1. Progress: The mobile layout and style is 98% done
2. Blockers: Very small mobile has issues with long titles
3. Availability: Thursday, Friday, Sunday
4. ETA: Sunday 1/31/21
![Screen Shot 2021-01-24 at 9 26 20 AM](https://user-images.githubusercontent.com/35665684/105635462-8780cc80-5e28-11eb-9db5-3b3bf4a8f961.png)
![Screen Shot 2021-01-24 at 9 26 35 AM](https://user-images.githubusercontent.com/35665684/105635466-89e32680-5e28-11eb-9d5f-7af1500eab07.png)

**Comment by trtincher** [comment]
@ExperimentsInHonesty It is unclear to me where to find the links for <a> tags and buttons.

**Comment by pawan92** [comment]
@trtincher  Could you please update:
Progress: Done except for links
Blockers: Unsure of where to find all the links
Availability: Weekday evenings and Sunday
ETA: 2/1/2021 (if blocker is addressed)

**Comment by trtincher** [comment]
Links for page:

- Bonnie should link my LinkedIn which is https://www.linkedin.com/in/bonnieawolfe/
- Send an email to civic opportunity team should go to civicopportunity@hackforla.org
- Change Volunteer Work on a project (each one should be a link to the appropriate filtered list)
- Partner with us is currently on the home page, so you can follow what is there (Patner with us is currently on the home page, so you can follow what is there)
- Help us Keep the Lights On - In-Kind services will need to go to donations@hackforla.org
- The donation form link goes to https://www.codeforamerica.org/donate

**Comment by trtincher** [comment]
@ExperimentsInHonesty Should I add the site link to the navbar to replace connect now. Or should the site be reviewed and live first?

**Comment by ExperimentsInHonesty** [comment]
@trtincher Please dont change the menu when you commit.  We will do that on a separate issue after we have approved of it live .

**Comment by trtincher** [comment]
Progress: Daniel's notes have all been addressed and ready for another review.
Blockers: all good now
Availability: Monday, Tuesday Night, Sunday
ETA: potentially 2/14/21 (Pending Reviews)

**Comment by tarang100** [comment]
Blockers: scss changes not being reflected in browsers.
ETA: Tuesday 2/23/19
Availability: Monday, Tuesday evening. Sunday.

**Comment by trtincher** [comment]
Working on final small edit
ETA: 3/2/21 at latest

**Comment by trtincher** [comment]
Blocker: There are styling issues on this page being affected by un-nested classes on another page. Waiting on #1158. 
ETA: 3/9/21

**Comment by tarang100** [comment]
It looks like #1158 has been merged. Please provide an update on the status or close if the issue has been completed

**Comment by trtincher** [comment]
PR was merged. Needs approval and review before adding link to the menu

**Comment by averdin2** [comment]
Other issues have been created to solve the rest of what needs to be completed with this page.
